### ISSUE TITLE
[Feature]: Time tracking

### ISSUE THREAD

**Issue by jseidel89** [mgmeyers/obsidian-kanban#672]
### Goal or desired outcome of this feature

It would be great to get an option to use time trackers on the cards itself. I tried the obsidian-time-tracker lines on the cards (https://github.com/daaru00/obsidian-time-tracker) but it does not seem to work inside tasks.

### Describe the feature

Having a time tracking button on each task from within the obsidian-kanban plugin would be awesome. One might also find a work-around to allow the use of existing tools such as obsidian-time-tracker on the cards.

### Can you think of any alternatives or work-arounds?

Alternatively, one might also have a list/column that actively tracks the time a card is in that list. So one could just drag the card over into the tracking list when starting the work and drag it back out once its done.

**Comment by kariyum** [comment]
Yes please, this one is a great feature to have. It will allow us to better estimate the completion time of recurrent tasks.

**Comment by TheSatanicFlame** [comment]
I vouch for this one too, Using it for school projects that require me to track time
